test/acceptance/pages.py

```diff
@@ -418,13 +418,13 @@ def num_completed(self):
             raise PageConfigurationError(msg)
 
         status_completed_css = self._bounded_selector(".step__status__value--completed")
-        status_current_css = self._bounded_selector(".step__status__number--current")
         complete_candidates = [int(x) for x in self.q(css=status_completed_css).text]
         if len(complete_candidates) > 0:
             completed = complete_candidates[0]
         else:
-            # The number completed if no longed available on this page, but can be inferred from the
+            # The number completed if no longer available on this page, but can be inferred from the
```

Comment:
Sorry to be so picky, but I still think there is a typo in this comment. Shouldn't it be "The number completed **is** no longer available..."